docs/targets/googlecloudstorage.md

```diff
@@ -0,0 +1,46 @@
+# Event Target for GoogleCloudStorage 
+This event target integrates with GoogleCloudStorage, using received Cloud Event messages to create Blob items.
+
+## Prerequisites
+
+* A storage bucket is expected to exist prior to running this event target. If you haven't already done so, create a bucket by following the instructions from the [Cloud Storage How-To Guides](https://cloud.google.com/storage/docs/how-to).
+
+* A [Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) is required to authenticate the event target and allow it to interact with Google. Create a [key in JSON format](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) for this service account and save it. It is required to be able to run an instance of the Google Cloud Storage event target.
```

Comment:
I am not a fan of "key in JSON format" could we simplify this?

Reply:
@daceynolan do you mean the way this is phrased, or the fact that it's JSON?

If it's the latter, the Google Cloud SDK unfortunately requires a full JSON here.

Reply:
@antoineco I meant the phrasing. I think the link is extremely long and just sounds weird reading it. I would recommend saying  "create a key" - key can be the link. Then I would say something like "This key should be in the format of JSON"

Reply:
I'm [guilty for this phrasing](https://github.com/triggermesh/docs/blob/9d36f563132ab3628745cd920d6bb1e9ae6531b3/docs/sources/googlecloudpubsub.md#service-account), Jeff just copied my mistake :)

The reason why I avoided "create a key" is because such key can be created in 2 different formats, so from my perspective of a non-native speaker, I felt that "create a key" sounded vague:

> Note: You can create service account keys in JSON or PKCS#12 (P12).
> -- https://cloud.google.com/iam/docs/creating-managing-service-account-keys

If the phrasing still feels wrong despite this context, I'll gladly go ahead and fix existing pages.

Reply:
how about 
``` A [Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)  along with a respective [key in JSON format](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) is required to authenticate the event target and allow it to interact with Google.```

Reply:
@JeffNeff Antoine is making this change. How about we make this the new verbiage?
```
 Create a [key][gc-iam-key] for this service account and save it. This key must be in JSON format. It is required to be
able to run an instance of the Google Cloud Pub/Sub event source.
```

Reply:
i think stating something is required under prerequisites is kinda redundant.. 

```
 Create a [key][gc-iam-key] for this service account and save it.
 ```
  seems like enough to me

Reply:
Right, in the page I linked the section is called "service account", that's why I wrote "the key is required ..." as in "that's something you'll have to enter in the form".